_episodes/20.md

```diff
@@ -0,0 +1,370 @@
+   the [Trino code style definition](https://raw.githubusercontent.com/airlift/codestyle/master/IntelliJIdea2019/Airlift.xml) 
+   that [you can import into Intellij](https://www.jetbrains.com/help/idea/copying-code-style-settings.html) 
+   to have the `Reformat Code` action to format in the desired style of the project.
+    
+1. Build the project.
+
+   One of the greatest resources in trino history is [this cheat sheet](https://gist.github.com/findepi/04c96f0f60dcc95329f569bb0c44a0cd)
+   created by [Piotr Findeisen](https://twitter.com/findepi). I use it for some
+   of the commands, but the most important use, is the "fast" build command he
+   adds on the top. In your terminal, make sure you are located in the root 
+   directory of the Trino project, and run the following command.
+   
+   ```
+   ./mvnw -pl '!:trino-server-rpm,!:trino-docs,!:trino-proxy,!:trino-verifier,!:trino-benchto-benchmarks' clean install \
+   -TC2 -nsu \
+   -DskipTests \
+   -Dmaven.javadoc.skip=true \
+   -Dmaven.source.skip=true \
+   -Dair.check.skip-all=true
+   ```
+   
+   This will build all necessary modules of the project to run almost everything
+   in Trino. The build excludes some modules, runs the compiler on multiple 
+   threads, skips the tests, javadocs, and the Airlift code style checks. If you
+   would like to run code style check on a specific module (e.g. 
+   trino-elasticsearch) then you can run the following command.
+   
+   ```
+   ./mvnw -pl ':trino-elasticsearch' clean install \
+   -TC2 -nsu \
+   -DskipTests \
+   -Dmaven.javadoc.skip=true \
+   -Dmaven.source.skip=true 
+   ```
+
+1. Sign the CLA.
+    
+    Sign the [contributor license agreement (CLA)](https://github.com/trinodb/cla/blob/master/Trino%20Foundation%20Individual%20CLA.pdf) 
+    to agree that all of your code you commit to the project is subject to the 
+    Apache License 2.0. Once you sign the agreement, scan and submit the form to
+    [cla@trino.io](mailto:cla@trino.io). This email gets checked every few days,
+    and you can check if your name has been added to the [contributors](https://github.com/trinodb/cla/blob/master/contributors)
+    list.
+    
+1. At this point you will want to look for an [issue labeled "good first issue"](https://github.com/trinodb/trino/issues?q=is%3Aopen+is%3Aissue+label%3A%22good+first+issue%22)
+This identifies issues that we think are more approachable for developers that 
+aren't as familiar with the Trino repository yet.
+
+1. One final thing before you move on to the contribution process. Before you
+start jumping in and changing the code, you'll also want to create a special
+branch for your changes. A branch in git makes a separate workflow for all the 
+changes you make to be isolated so that if something goes wrong, or you need to 
+compare with an older branch you can do so. The default branch may either be
+named `master` or `main`. Due to the connotation of the word `master` with 
```

Comment:
remove the whole master vs main since it is not relevant here

Reply:
I guess...I do think this is useful context to understand that other open source projects' master branch is called main and why. But for Trino it doesn't matter I guess.